github/ipfs.yml

```diff
@@ -5686,6 +5686,18 @@ teams:
         - ntninja
         - whereswaldon
     privacy: closed
+  ipfs-devrel:
```

Comment:
If we go with just devrel, then we could mention it with @ipfs/devrel
```suggestion
 devrel:
```

Completely up to you, let me know :)

Reply:
much better :)